src/field/overlay_01_apricorn_tree_anim.c

```diff
@@ -180,16 +180,16 @@ static BOOL Task_AnimApricornTree(TaskManager *taskman) {
 }
 
 static LocalMapObject *CreateJumpingApricornObj(MapObjectManager *taskman, u32 sprite, u32 x, u32 z) {
-    LocalMapObject *obj = CreateSpecialFieldObject(taskman, x, z, 0, sprite, 0, 1);
+    LocalMapObject *obj = MapObject_Create(taskman, x, z, 0, sprite, 0, 1);
     GF_ASSERT(obj != NULL);
 
     MapObject_SetID(obj, obj_apricorn);
     MapObject_SetType(obj, 0);
     MapObject_SetFlagID(obj, 0);
     MapObject_SetXRange(obj, -1);
     MapObject_SetYRange(obj, -1);
-    MapObject_SetFlagsBits(obj, MAPOBJECTFLAG_UNK13 | MAPOBJECTFLAG_UNK10);
-    MapObject_ClearFlagsBits(obj, MAPOBJECTFLAG_UNK8 | MAPOBJECTFLAG_UNK7);
+    MapObject_SetFlagsBits(obj, (MapObjectFlagBits)(MAPOBJECTFLAG_UNK13 | MAPOBJECTFLAG_UNK10));
```

Comment:
is it really worth having the enum if you have to keep casting it? #define could work just as well

Reply:
for now, yeah, I think it's fine

Reply:
I might move this to a define later, but not here/now